types/meteor/accounts-base.d.ts

```diff
@@ -31,6 +33,16 @@ declare module 'meteor/accounts-base' {
             callback?: (error?: Error | Meteor.Error | Meteor.TypedError) => void,
         ): string;
 
+        function createUserAsync(
+            options: {
+                username?: string | undefined;
+                email?: string | undefined;
+                password?: string | undefined;
+                profile?: Object | undefined;
```

Comment:
Both `Object` and `object` aren't restrictive enough. `Record<string, unknown>` would be safer here.

Reply:
Hmm. This is just copy-pasted from the definition of `createUser` above. Can we merge this as-is in the interests of keeping those two in sync, and keeping this in sync with the Meteor core definitions, and I'll follow up with another PR to fix the uses of `Object` across the board in the type definitions?

Reply:
OK, sure.